### PR TITLE
rpi-cmdline: Add option to append isolcpus to cmdline.txt

### PR DESCRIPTION
**Pull request by ajpenner**
**- What I did**
I have added a new variable to be parsed and added to cmdline.txt that allows us to take advantage of the isolcpus capabilities typically used for real time tasks

**- How I did it**
I added a new variable to be read from conf/local.conf and added to cmdline.txt via the rpi-cmdline.bb recipe

### CODE REVIEW

File: recipes-bsp/bootfiles/rpi-cmdline.bb [hunk 1]
```diff
@@ -29,6 +29,19 @@ CMDLINE_LOGO ?= '${@oe.utils.conditional("DISABLE_RPI_BOOT_LOGO", "1", "logo.nol
 # to enable kernel debugging.
 CMDLINE_DEBUG ?= ""
 
+# Add means to isolate processors for realtime setups
+# ISOLATED_CPUS may have the form of a comma separated
+# list of processor numbers "0,1,3" or a range "0-2"
+# a combination of the two "0-1,3", or a single processo
```
Comment (kraj) [inline recipes-bsp/bootfiles/rpi-cmdline.bb:35]: typo at the end of line

Comment (ajpenner) [reply recipes-bsp/bootfiles/rpi-cmdline.bb]: Fixed

**Review by agherzan — DISMISSED** [review]

File: recipes-bsp/bootfiles/rpi-cmdline.bb [hunk 2]
```diff
@@ -29,6 +29,19 @@ CMDLINE_LOGO ?= '${@oe.utils.conditional("DISABLE_RPI_BOOT_LOGO", "1", "logo.nol
 # to enable kernel debugging.
 CMDLINE_DEBUG ?= ""
 
+# Add means to isolate processors for realtime setups
```
Comment (agherzan) [inline recipes-bsp/bootfiles/rpi-cmdline.bb:32]: What do you mean by the above?

Comment (ajpenner) [reply recipes-bsp/bootfiles/rpi-cmdline.bb:32]: I could have written this as "Add a configuration variable to isolate processors..." which has the same intent. I am attempting to describe the purpose of the ISOLATED_CPUS variable

Comment (agherzan) [reply recipes-bsp/bootfiles/rpi-cmdline.bb:32]: I see. I would rephrase that as it was confusing when I first read it.

Comment (ajpenner) [reply recipes-bsp/bootfiles/rpi-cmdline.bb]: Fair point, the change has been made

**Review by agherzan — APPROVED** [review]

**Review by agherzan — APPROVED** [review]
